definitions/fenix.toml

```diff
@@ -337,46 +337,57 @@ select_expression = """
           AND event.name = 'contile_click'
       ),0)
 """
-data_source = "events" 
+data_source = "events"
 friendly_name = "Sponsored Tiles Clicks"
 description = "Number of times user clicked a Contile Sponsored Tile."
 owner = "xluo-all@mozilla.com"
 
 [metrics.spoc_tiles_disable_rate]
-select_expression = """   
+select_expression = """
   COALESCE(MAX(
     CAST(
        metrics.boolean.customize_home_contile AS int )
   ),0)
 """
-data_source = "metrics" 
+data_source = "metrics"
 bigger_is_better = false
 friendly_name = "Sponsored Tiles Disable Rate"
 description = "Fraction of users who disable Contile Sponsored Tiles"
 owner = "xluo-all@mozilla.com"
 
 [metrics.spoc_tiles_preference_toggled]
-select_expression = """   
+select_expression = """
   COALESCE(SUM(CASE WHEN
           event.category = 'customize_home'
           AND event.name = 'preference_toggled'
           AND `mozfun.map.get_key`(event.extra, 'preference_key') = 'contile'
 	THEN 1 ELSE 0 END
   ),0)
 """
-data_source = "events" 
+data_source = "events"
 bigger_is_better = false
 friendly_name = "Sponsored Tiles Preference Toggled"
 description = "Number of times Contile Sponsored Tiles setting is flipped."
 owner = "xluo-all@mozilla.com"
 
 [metrics.new_profile_activation]
 select_expression = "COALESCE(SUM(activated))"
-data_source = "new_profile_activation" 
+data_source = "new_profile_activation"
 friendly_name = "New Profile Activation"
 description = "A new profile is counted as activated one week after creation if it meets the following conditions: 1) at least 3 days of use during first week 2) at least one search between days 4-7."
 owner = "vsabino@mozilla.com"
 
+[metrics.fxa_sign_in_rate]
+select_expression = """
+  COALESCE(MAX(
```

Comment:
If this is a rate, should we be using `AVG` instead of `MAX`? Could you alternatively use:
```
AVG(IF(metrics.boolean.preferences_signed_in_sync, 1, 0))
```

Reply:
Ah, that makes perfect sense. I guess one complication with Metric Hub is that it's difficult to know from the definition alone how the metric is expected to be aggregated and/or who the intended consumers are.

If we expected this to be used in a SQL query and grouped by OS/country/release_channel, I think my suggestion would make sense. But if the metric is expected to be used in experiments and aggregated by `client_id` then using `MAX` here is makes sense. I think these metrics aren't really "rates" in the way they're defined, but they are "rates" in the way they're used, which a bit confusing. Probably not a problem to solve in this PR though.

I _think_ that `MAX(IF(metrics.boolean.preferences_signed_in_sync, 1, 0))` is equivalent to the existing pattern (and is more readable imo), but maintaining consistency is also valuable; I don't have a strong preference either way.